mindsdb_native/libs/helpers/text_helpers.py

```diff
@@ -158,10 +158,8 @@ def is_foreign_key(data, column_name, data_subtype, other_potential_subtypes):
             is_uuid = True
             is_same_length = True
 
-            for char in str(val):
-                if char not in ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
-                                'a', 'b', 'c', 'd', 'e', 'f', '-']:
-                    is_uuid = False
+            uuid_charset = set('0123456789abcdef-')
+            set(val).issubset(uuid_charset)
```

Comment:
when i train 'home_rentals' i get error in this row:
`ERROR:mindsdb-logger-1cc370aa-dbab-11ea-84a9-2c56dc4ecd27:/home/maxs/dev/mdb/venv_new/sources/mindsdb_native/mindsdb_native/libs/controllers/transaction.py:178 - 'float' object is not iterable`
it try execute `set(1906.0)` and fail

Reply:
it should be `set(str(val)).issubset(uuid_charset)`